scopes/lanes/lanes/lane.cmd.ts

```diff
@@ -406,15 +413,15 @@ export class LaneImportCmd implements Command {
 }
 
 export class LaneCmd implements Command {
-  name = 'lane [lane-name]';
+  name = 'lane [lane-name]'; // @david is the lane-name argument required?
```

Comment:
it used to be mandatory until 4-5 weeks ago. then we moved to default to current lane, if you didn't pass any argument.

Reply:
It looks like if there's no lane then this just defaults to the behaviour of `bit lane list`
Is there any reason we have this plain `bit lane` command?
![image](https://github.com/teambit/bit/assets/67269267/b13961b8-dfb5-4199-8fce-7514d3f0ef71)

Reply:
Correct. `bit lane` defaults to `bit lane list`.  You can change the `[lane-name]` so maybe `[subcommand]` or just remove it.

Reply:
ok great, will update and resolve this convo